python/vineyard/data/utils.py

```diff
@@ -144,15 +147,30 @@ def ensure_ipc_client(client, error_message=None):
 def build_buffer(
     client, address, size, *args, **kwargs
 ) -> Union["Object", "ObjectMeta", "ObjectID"]:
-    '''Build a blob in vineyard server for the given bytes or memoryview.
+    '''Build a blob or a remote blob in vineyard server
+        for the given bytes or memoryview.
 
     If address is None or size is 0, an empty blob will be returned.
     '''
-    ensure_ipc_client(
-        client,
-        "Vineyard RPC client cannot be used to create local blobs, "
-        "try using an IPC client or `rpc_client.create_remote_blob()`",
-    )
+    if isinstance(client, RPCClient):
+        # copy the address with size to a local payloads
+        if size == 0 or address is None:
+            meta = ObjectMeta()
+            meta.id = ObjectID("o8000000000000000")
```

Comment:
Don't hard code the empty blob in Python.

Like `client.create_empty_blob`, you could add such a method to RPC client as well, if needed.